src/main/java/com/squareup/javapoet/AnnotationSpec.java

```diff
@@ -122,6 +122,9 @@ public static Builder builder(Class<?> type) {
   public Builder toBuilder() {
     Builder builder = new Builder(type);
     builder.members.putAll(members);
+    for (Map.Entry<String, List<CodeBlock>> entry : builder.members.entrySet()) {
```

Comment:
I think I'd have rather looped over the `members`' entry set to `put()` each entry into `builder.members` instead of using `putAll` and then replacing the entries' values.